src/streamlink_cli/argparser.py

```diff
@@ -146,6 +146,12 @@ def hours_minutes_seconds(value):
     :param value: hh:mm:ss
     :return: seconds
     """
+    try:
+        if int(value):
```

Comment:
I think you could just do `return int(value)` here, normally it returns an `int` and 0 would be a valid result too.